docs/interpreter/pig.md

```diff
@@ -29,47 +29,39 @@ which in turns enables them to handle very large data sets.
     - No pig alias in the last statement in `%pig.query` (read the examples below).
     - The last statement must be in single line in `%pig.query`
     
-## Supported runtime mode
-  - Local
-  - MapReduce
-  - Tez_Local (Only Tez 0.7 is supported)
-  - Tez  (Only Tez 0.7 is supported)
-  - Spark_Local (Only Spark 1.6.x is supported, by default it is Spark 1.6.3)
-  - Spark (Only Spark 1.6.x is supported, by default it is Spark 1.6.3)
 
 ## How to use
 
-### How to setup Pig
+### How to setup Pig for different modes
 
 - Local Mode
 
-    Nothing needs to be done for local mode
+    Set `zeppelin.pig.execType` as local.
 
 - MapReduce Mode
 
-    HADOOP\_CONF\_DIR needs to be specified in `ZEPPELIN_HOME/conf/zeppelin-env.sh`.
+    Set `zeppelin.pig.execType` as mapreduce. HADOOP\_CONF\_DIR needs to be specified in `ZEPPELIN_HOME/conf/zeppelin-env.sh`.
```

Comment:
perhaps put `mapreduce` in quotes? `"mapreduce"`? I don't know if it's more confusing that way (ie. that people will type qoute)